sw/device/tests/sim_dv/sram_ctrl_scrambled_access_test.c

```diff
@@ -6,6 +6,7 @@
 #include <stdint.h>
 
 #include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_rstmgr.h"
 #include "sw/device/lib/runtime/log.h"
```

Comment:
Why are these two C tests moved to sim_dv? It does not seem like these tests need external stimulus that can only be generated by the DV testbench.

Reply:
I speculate they will need to be DV once a backdoor check is implemented.